source/cloud/aws/sagemaker.md

```diff
@@ -41,24 +41,16 @@ set -e
 
 sudo -u ec2-user -i <<'EOF'
 
-mamba create -y -n rapids -c rapidsai -c conda-forge -c nvidia rapids=24.12 python=3.12 cuda-version=12.4 \
-    boto3 \
-    ipykernel \
-    'sagemaker-python-sdk>=2.239.0'
+eval "$(conda shell.bash hook)"
```

Comment:
Why was this necessary? If `conda` comes pre-installed in the AMI I'd expect it to already be activated when you log in as the default user.

e.g. I'd expect `conda {cmd}` to "just work".